MyBooking/Views/BookingView/README.md

```diff
@@ -0,0 +1,25 @@
+## Available Scripts
+
+In the 'Views/BookingView' directory, you can run:
+
+### `npm install`
+
+Installs dependensis for the UI part of project
+
+### `npm start`
+
+Runs the app in the development mode.<br />
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
```

Comment:
our api runs on 49561 or am i wrong?

Reply:
You are right, but I think it would be better to run the app on the one port and call to another, it would be easier not to confuse and clearly understand that we can get our data in the one place